MetaGenerator/DotnetToolInstaller.cs

```diff
@@ -40,6 +37,14 @@ public void InstallTool(string path, string toolName, string? version)
             dotnet.Execute(GetToolInstallArguments(toolName, version), path);
         }
 
+        private static bool HasDotNetToolManifest(string path)
+        {
+            const string dotnetToolsFileName = "dotnet-tools.json";
+            var hasDotnetToolManifest = Directory.GetFiles(path, dotnetToolsFileName, SearchOption.AllDirectories)
+                .Any();
+            return hasDotnetToolManifest;
```

Comment:
return this directly

Reply:
Addressed in: [Return HasDotnetToolManifest directly](https://github.com/messerli-informatik-ag/meta-generator/pull/24/commits/ed06eda13e779b0a4bb1448479145e1830fb3741)